spec/serializers/federation_entity_serializer_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe FederationEntitySerializer do
+  class TestEntity < DiasporaFederation::Entity
+    property :test, :string
+  end
+
+  let(:object) { double }
+
+  def enable_test_entity_builder
+    allow(Diaspora::Federation::Mappings).to receive(:builder_for).with(object.class).and_return(:test_entity)
+    allow(Diaspora::Federation::Entities).to receive(:test_entity).with(object) {
+      TestEntity.new(test: "asdf")
+    }
+  end
+
+  before do
+    enable_test_entity_builder
```

Comment:
Why did you extract this to a method? If it's only used here I would just move the code here.